Source/Scene/ModelExperimental/PickingPipelineStage.js

```diff
@@ -34,15 +36,18 @@ PickingPipelineStage.process = function (
   var context = frameState.context;
   var runtimeNode = renderResources.runtimeNode;
   var shaderBuilder = renderResources.shaderBuilder;
+  var model = renderResources.model;
 
   shaderBuilder.addDefine("USE_PICKING");
 
-  if (defined(runtimeNode.node.instances)) {
+  if (defined(model._featureTable)) {
+    // For models with features, the pick texture is used.
+    processPickTexture(renderResources, model._featureTable);
+  } else if (defined(runtimeNode.node.instances)) {
```

Comment:
Are we supporting instancing + feature metadata for feature tables in this PR?

This is part of the spec:
https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_feature_metadata#feature-id-instance-attributes

Reply:
Support for that has been added.